<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_calendar</class>
 <widget class="QWidget" name="w_calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>969</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QCalendarWidget" name="calendarDec">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>720</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>162</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarAug">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>560</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarJul">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>560</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarJan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarFev">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarJun">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarMai">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollAreaHeader">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>741</width>
     <height>331</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>1000</width>
     <height>200</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>739</width>
      <height>329</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCalendarWidget" name="calendarSep">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarMar">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarOct">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>720</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarApr">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>400</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarNov">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>720</y>
     <width>240</width>
     <height>148</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Details of Mets</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Issue number as text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Issue number as parsed :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>134</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pages :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Supplements :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No issue selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbIssueNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No issue selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbIssueParsed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No issue selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbPages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No issue selected</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listSupplement">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>201</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listMets">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbYear">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select year :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>880</y>
     <width>671</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Add Comment</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remarks on date :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddComment">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtComment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>301</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateComment">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnViewMets">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Directory</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>890</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
